### PR TITLE
use the new run.sh script in rbenchmarking repo

### PR DESCRIPTION
**Pull request by o-**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -78,7 +78,11 @@ test_flaky:
...
+    - git clone https://github.com/charig/ReBench.git -b envVarsSupport && cd ReBench
+    - pip install --user .
+    - export PATH=$HOME/.local/bin:$PATH
+    - cd ..
+    - git clone https://github.com/reactorlabs/rbenchmarking
     - cd build_release
-    - ../tools/downloadBenchs.sh ../build_release
-    - ../tools/runBenchs.sh --installRebench --travis
-    - PIR_DEOPT_CHAOS=1 ../tools/runBenchs.sh --travis
+    - ../rbenchmarking/run.sh bin ../external/custom-r/bin --warmup 1 e:RIR e:PIR -r -SN --iterations 2
```
Comment (charig) [inline .gitlab-ci.yml]: Why do we need warmup for testing?

**Review by charig — APPROVED** [review]
Modulo some small changes to RBenchmarking so that it also work on macOS

File: .gitlab-ci.yml (hunk 1 above)
Comment (charig) [reply .gitlab-ci.yml]: RBenchmarking updated: here an example of the slightly different command from travis https://github.com/reactorlabs/RBenchmarking/blob/master/.travis.yml#L89

**Review by charig — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (o-) [reply .gitlab-ci.yml]: why is it in Setup?

Comment (charig) [reply .gitlab-ci.yml]: My original idea was that the setup scripts are not strictly necessary but auxiliary. You have a rebench.conf in the root dir and you can run it manually if you know where the implementations are. That said, I have no strong opinions about it, feel free to move it elsewhere.

**Pull request merged**